src/js/live-tracker.js

```diff
@@ -90,6 +94,21 @@ class LiveTracker extends Component {
 
     this.on(this.player_, 'play', this.trackLive_);
     this.on(this.player_, 'pause', this.trackLive_);
+    this.on(this.player_, 'firstplay', this.handleFirstplay);
```

Comment:
we should not be using `firstplay`. It's non-standard, deprecated, and I'm not sure it's complete.

Reply:
done, I didn't realize that it wasn't something we fired internally.